### ISSUE TITLE
3.2.0 to 3.3.1

### ISSUE THREAD

**Issue by ryanregazo** [UnamSanctam/SilentCryptoMiner#1811]
Hello Unam, what happen if I execute the 3.3.1 version into a 3.2.0 infected PC without uninstall it first?

**Comment by DerXanRam** [comment]
Both will work.

**Comment by UnamSanctam** [comment]
Yes as @DerXanRam said, both with start and run but if both had the same "Startup" settings then the new miner file will replace the old miner file. So on the next restart only the new miner should start.
